backend/spacewalk-backend.spec

```diff
@@ -83,6 +83,7 @@ BuildRequires:  python3-debian
 BuildRequires:  python3-rhn-client-tools
 BuildRequires:  python3-rhnlib >= 2.5.74
 BuildRequires:  python3-rpm
+BuildRequires:  python3-rpm-macros
```

Comment:
The package name is `python-rpm-macros` and it's used for all python flavors, `python3-rpm-macros` is specified as `Provides` in the package. Maybe better to use `BuildRequires: python-rpm-macros`?

Reply:
I keep this as we use this already somewhere and that build also on AlmaLinux. I am not sure if SUSE and AlmaLinux behave the same.

Reply:
Hmm, it seems Alma really has separate `python-rpm-macros` and `python3-rpm-macros` and the packages are different.